src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -135,6 +135,25 @@ def skip_table_or_view(self, schema_name: str, table_name: str, load_table: Call
         except BadRequest as err:
             logger.error(f"Failed to apply skip marker for Table {schema_name}.{table_name}: {err!s}", exc_info=True)
 
+    def unskip_table_or_view(self, schema_name: str, table_name: str, load_table: Callable[[str, str], Table | None]):
+        # Removes skip mark from the table property
+        try:
+            table = load_table(schema_name, table_name)
+            if table is None:
+                raise NotFound("[TABLE_OR_VIEW_NOT_FOUND]")
```

Comment:
Should this be in the `try-except`?

Reply:
This one is tricky yes it should be but when I made the change in skip function I realised the test `test_skip_missing_table` fails as apparently `NotFound("[TABLE_OR_VIEW_NOT_FOUND]")` is not found... For now I keep it inside `try-except` and create a unit test for unskip when the table is None.

![image](https://github.com/user-attachments/assets/ffee347f-3043-4a3b-adeb-c04fbe809991)

Reply:
yes, keep it in for now. I would liked separate `try-except` for loading the table and executing the SQL, this is good for now

Reply:
Done!